packages/web/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Any way to share common config options like babel (where you can just have a core config, grab it, spread it in and change what you need)? 

Write our own extends?

Reply:
For sure, let me try to fix this.